zinit-install.zsh

```diff
@@ -1447,25 +1447,25 @@ builtin source "${ZINIT[BIN_DIR]}/zinit-side.zsh" || {
         local url=https://$urlpart
```

Comment:
Changes in this file seem unrelated. Looks like `bpick` work and work from other PRs. Can you unstage these?

Reply:
Correct. Snuck in from another branch. This code belongs in #201.

I have removed it.